Update MsgFee spec to account for new events added

nullpointer0x00 opened the pull request:
<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺
v                               ✰  Thanks for creating a PR! ✰
v    Before smashing the submit button please review the checkboxes.
v    If a checkbox is n/a - please still include it but + a little note why
☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->

## Description

Update msg fees event spec with details about new events in 1.12+ 

closes: #1091 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit and integration [tests](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

arnabmitra commented on x/msgfees/spec/05_events.md:
```diff
@@ -8,18 +8,17 @@ Existing fee event continue to show total fee charged
+| message  | min_base_fee  | the minimum fee is the fee that will always be charged on failure or success                                     |
+| message  | basefee       | base fee is the total amount charged for a successful transaction.  This will be equal to fee - additionalfees   |
 
 ## Tx with Additional Fee
```
Tx with Additional Fee will have fee, min_base_fee, basefee also?

nullpointer0x00 replied:
I updated this section with your suggestions.

SpicyLemon commented:
I added an update to #1060 to fix this documentation, so I'm not sure if this PR is needed.

nullpointer0x00 commented:
@SpicyLemon I added a comment to your PR.  It makes sense to put them in there and make things easier.  I added a suggestions to add more details and move an event to a different section.

Closing this PR since it is moved to PR #1060
